stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/WebSocketUtil.java

```diff
@@ -121,4 +124,21 @@ public static void populateEndpoint(Session session, BStruct endpoint) {
         endpoint.setBooleanField(0, session.isSecure() ? 1 : 0);
         endpoint.setBooleanField(1, session.isOpen() ? 1 : 0);
     }
+
+    public static BValue[] getWebSocketError(Context context, ChannelFuture future, String message, String hash)
+            throws InterruptedException {
+        AtomicReference<BValue[]> error = new AtomicReference<>();
+        future.addListener((ChannelFutureListener) future1 -> {
```

Comment:
Since there can be many futures shall we give specific name to this future variable.